rcon/webhook_service.py

```diff
@@ -178,6 +182,18 @@ class QueueParts:
     id: str
 
 
+@dataclass
+class ScoreboardQueueParts:
```

Comment:
Same as above: I would love a more generic approach. The webhook service should not need to know about specific types of messages or consumers of it's api.